README.md

```diff
@@ -260,36 +177,121 @@ quantized_model = nncf.quantize(model, calibration_dataset)
 
 </details>
 
+
+[//]: # (NNCF provides full  [samples]&#40;#post-training-quantization-samples&#41;, which demonstrate Post-Training Quantization usage for PyTorch, TensorFlow, ONNX, OpenVINO.)
+
+### Training-time Compression
+
+The basic workflow steps:
+1) Create an NNCF config object from (a) JSON configuration file and (b) dataloader for the compression initialization.
+2) Pass your model together with the config to the `create_compressed_model` function which returns two entities:
+   1) Compression controller — an object allowing you to control the compression, for example during training.
+   2) Compressed model which may further be used as a regular PyTorch model for fine-tuning and inference.
+
```

Comment:
It looks like you forgot the following steps:
- model fine-tuning
- model export

Reply:
Thanks, done.